src/components/widgets/StoicWidget/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import { useGetStoicQuote } from "./useGetStoicQuote";
+import { ReactComponent as GreekLogo } from "../../../assets/icons/greek.svg";
+import { ReactComponent as CloudLogo } from "../../../assets/icons/cloud.svg";
+import { ReactComponent as CloudAltLogo } from "../../../assets/icons/cloud-alt.svg";
+
+interface StoicData {
+  author: string;
+  quote: string;
+}
+
+export const StoicWidget = () => {
+  const [stoicData, setStoicData] = useState<StoicData[] | undefined>([]);
+  useGetStoicQuote(setStoicData);
+
+  const randomNumber = Math.floor(Math.random() * 10);
+
+  return (
+    <React.Fragment>
+      <div className="stoic-widget__wrapper">
+        <GreekLogo />
+        <div className="quote__container">
+          {stoicData && <p>"{stoicData[randomNumber]?.quote}"</p>}
+          {stoicData && <h6>- {stoicData[randomNumber]?.author}</h6>}
+        </div>
+        <div className="clouds__wrapper">
+          <div className="cloud-b__container">
+            <div className="inner-cloud-b__container">
+              <CloudAltLogo transform="scale (-1 1)" />
```

Comment:
Since a logo is something that represents an organization naming these "logo" is technically wrong. Its really just an icon.